django_teams/templates/django_teams/team_detail.html

```diff
@@ -12,7 +12,7 @@ <h1>{{ object }} <small>detailed information</small></h1>
             <h3 class="panel-title">Teachers</h3>
```

Comment:
NotImplementedError at /teams/4/
DISTINCT ON fields is not supported by this database backend